packages/samples/headless-ssr-commerce/package.json

```diff
@@ -12,6 +12,7 @@
   },
   "dependencies": {
     "@coveo/headless-react": "2.1.1",
+    "jsdom": "25.0.1",
```

Comment:
dev deps. Don't want implementers to think that jsdom is needed for this sample to work.

Reply:
yep, fixed